src/app-layout/__integ__/mobile.test.ts

```diff
@@ -0,0 +1,106 @@
+
+const wrapper = createWrapper().findAppLayout();
+
+describe.each(['classic', 'visual-refresh', 'visual-refresh-toolbar'] as const)('%s', theme => {
+  class MobileTestingPage extends BasePageObject {
+    async visit(pageName: string) {
+      const params = new URLSearchParams({
+        visualRefresh: `${theme.startsWith('visual-refresh')}`,
+        appLayoutToolbar: `${theme === 'visual-refresh-toolbar'}`,
+      });
+      await this.browser.url(`#/light/app-layout/${pageName}?${params.toString()}`);
+      await this.waitForVisible(wrapper.toSelector());
+    }
+
+    isContentClickable() {
+      // cannot assert on findContentRegion selector, because it has `display:contents` and fails to detect as clickable
+      return this.isClickable(wrapper.findContentRegion().findContainer().toSelector());
+    }
+
+    getViewportWidths() {
+      return this.browser.execute(() => [document.documentElement.scrollWidth, window.innerWidth]);
+    }
+  }
+
+  function setupTest(testFn: (page: MobileTestingPage) => Promise<void>) {
+    return useBrowser(viewports.mobile, async browser => {
+      const page = new MobileTestingPage(browser);
+      await testFn(page);
+    });
+  }
+
+  describe.each([
+    'default',
+    'disable-paddings',
+    'multi-layout-simple',
+    'with-split-panel',
+    'with-table',
+    'with-wizard-and-table',
+  ])('%s', pageName => {
+    test(
+      'should not render horizontal scrollbar on mobile',
+      setupTest(async page => {
+        await page.visit(pageName);
+        const [documentWidth, windowWidth] = await page.getViewportWidths();
+        expect(documentWidth).toEqual(windowWidth);
+      })
+    );
+  });
+
+  test(
+    'main content is clickable by defaults',
+    setupTest(async page => {
+      await page.visit('with-fixed-header-footer');
+      await expect(page.isContentClickable()).resolves.toEqual(true);
+      await expect(page.isClickable(wrapper.findNavigation().toSelector())).resolves.toEqual(false);
+      await expect(page.isClickable(wrapper.findTools().toSelector())).resolves.toEqual(false);
+      await expect(page.isClickable('#h')).resolves.toEqual(true);
+      await expect(page.isClickable('#f')).resolves.toEqual(true);
+    })
+  );
+
+  test(
+    'header and footer should be clickable when nav panel is open',
+    setupTest(async page => {
+      await page.visit('with-fixed-header-footer');
+      await page.click(wrapper.findNavigationToggle().toSelector());
+      await expect(page.isContentClickable()).resolves.toEqual(false);
+      await expect(page.isClickable(wrapper.findNavigation().toSelector())).resolves.toEqual(true);
+      await expect(page.isClickable('#h')).resolves.toEqual(true);
+      await expect(page.isClickable('#f')).resolves.toEqual(true);
+    })
+  );
+
+  testIf(theme !== 'visual-refresh-toolbar')(
+    'header and footer should be clickable when nav panel is open in scrolled to the bottom state',
+    setupTest(async page => {
```

Comment:
Fixing this and the test below requires a separate PR on its own. 

It has something to do with this line and needs a separate investigation: https://github.com/cloudscape-design/components/blob/6c9f53017157c1ad8d1364c741bab014a6ef951f/src/app-layout/visual-refresh-toolbar/skeleton/styles.scss#L96